CONTRIBUTING.md

```diff
@@ -33,99 +30,111 @@ If you use [nvm](https://github.com/creationix/nvm) or [nave](https://github.com
+If you want to add a new component:
 
-### Sass mixins
+1. Use `bpk-component-boilerplate` to create a new skeleton React component
+2. It's unlikely you will need to create a class component. If you are in doubt, have a look at the React guidelines for component and see if you can use a function component instead: [React component guidelines](https://react.dev/reference/react/Component)
+3. Create stories - each component has a set of stories living under `examples/bpk-component-{name}/stories.ts`. Stories should cover most visual variants of a component. Read more about Storybook stories [here](https://storybook.js.org/docs/react/writing-stories/introduction)
+4. Create tests
+    - Visual regression tests - Each UI component's stories should also include a story that begins with the name `VisualTest` - these will then be picked up by Percy to run on CI
+    - Unit tests - Unit tests live in the same folder with the component's code and rely on `jest` and `React Testing Library`
+5. Update `README.md` following the boilerplate format
 
-All Sass mixins are defined in the [`bpk-mixins`](https://github.com/Skyscanner/backpack-foundations/tree/main/packages/bpk-mixins) package. The package also exposes the Sass variables from the [`@skyscanner/bpk-foundations-web`](https://github.com/Skyscanner/backpack-foundations/tree/main/packages/bpk-foundations-web) package.
+#### Contribute breaking changes
 
-If you add a new file of mixins, for example for a new *atom*, make sure you add the file to the imports in `_index.scss`.
+Anytime a change could break existing applications it's considered a breaking change. To make upgrading Backpack easier for consumers, breaking changes should follow a deprecation cycle.
 
-### React component
+In most cases, it is recommended to create a V2 component and provide a migration guide. If your breaking change either
+- requires consumers to make multiple modifications to their code to adopt the changes (e.g. renaming or removing multiple props of an API), OR
+- involves a significant alteration of the structure or API of a component in a way that keeping both APIs within one component may make the code unreadable (e.g. rewriting a component to reduce the number of files from 20 to 5)
+then you should contribute your changes as a separate V2 component.
 
-**All new components should be written in Typescript.**
+Once the old component has been removed from the codebase and is no longer in use, the Koala team will run automated scripts to make the new component the default.
 
-Use `bpk-component-boilerplate` to create a new skeleton React component. Once this is created, use existing components for code style inspiration.
+If your breaking change doesn't require consumers to make many modifications or its usages are limited (around 10-20 usages across Skyscanner web repositories), and keeping both versions in one component doesn't affect the readability of it, you will not need to create a separate component.
```

Comment:
10-20 usages seems like a lot so wondering if we should be quoting such a high number, this is what would prompt me to make a separate component as that is a massive impact

Reply:
It's still a large impact especially if its across multiple projects, yes the numbers are more significant but its still a lot of places that people will have to plan to adopt